pykern/pksetup.py

```diff
@@ -342,7 +342,18 @@ def main():
     Args:
         kwargs: see `setuptools.setup`
     """
+    def _assert_package_versions():
+        """Raise assertion if another module has installed incompatible version of setuptools
+        """
+        from packaging.version import Version
+
+        assert Version(setuptools.__version__) < Version('57'), \
+            'Some package installed a newer version of setup tools that does not work with pykern.pksetup'
+
     name = kwargs['name']
+    if name != 'pykern':
+        # POSIT: setup.py has 'setuptools<57'
```

Comment:
This comment should be moved to _assert_package_versions docstring.

Reply:
Just move it into the docstring like this
```python
 def _assert_package_versions():
        """Raise assertion if another module has installed incompatible version of setuptools
            POSIT: setup.py has 'setuptools<57'
        """
```
or replace the docstring? ei.
```python
 def _assert_package_versions():
        """POSIT: setup.py has 'setuptools<57'"""
```

Reply:
Move the string and add some space and formatting:

```
"""Raise assertion if another module has installed incompatible version of setuptools

POSIT: setup.py has 'setuptools<57'
""""
```

Reply:
You got it 👍